certified-connectors/Rainbird/apiDefinition.swagger.json

```diff
@@ -0,0 +1,865 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Rainbird is a no-code intelligent automation platform that enables you to automate complex decisions. This connector enables you to interact with knowledge maps you\u2019ve built in Rainbird.",
+    "contact": {
+      "name": "Rainbird Technologies Ltd",
+      "url": "https://rainbird.ai/contact-us/",
+      "email": "support@rainbird.zohodesk.eu"
+    },
+    "title": "Rainbird",
```

Comment:
@Mat-Rig the version and title properties should be above description, please move them. You can refer to the [apiDefinition.swagger.json template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json).